components/runners/ambient-runner/ambient_runner/platform/auth.py

```
@@ -533,6 +562,43 @@ async def populate_mcp_server_credentials(context: RunnerContext) -> None:
             logger.warning(f"Failed to fetch MCP credentials for {server_name}: {e}")
 
 
+_GH_WRAPPER_DIR = "/tmp/bin"
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/bin"

(S108)